orc8r/cloud/go/obsidian/handler.go

```diff
@@ -227,7 +229,16 @@ func GetNetworkId(c echo.Context) (string, *echo.HTTPError) {
 	if nid == "" {
 		return nid, NetworkIdHttpErr()
 	}
-	return nid, CheckNetworkAccess(c, nid)
+	sanNid := sanitizeString(nid)
+	return sanNid, CheckNetworkAccess(c, sanNid)
```

Comment:
why does nid needs to be sanitized here? why is `nid` comming with `\n` or `\r`?